packages/vkui/src/components/Pagination/Pagination.tsx

```diff
@@ -72,6 +80,8 @@ export interface PaginationProps extends Omit<HTMLAttributesWithRootRef<HTMLElem
    */
   getPageLabel?(isCurrent: boolean): string;
   onChange?(page: number): void;
```

Comment:
```suggestion
  onChange?(page: number, event: React.MouseEvent<HTMLElement>): void;
```

давай ещё добавим будем передавать `event` (надо будет ещё в `PaginationNavigationButtonProps` тип поправить), чтобы можно было делать `event.preventDefault()` в едином месте.

Reply:
готово